src/AstAbstract.h

```diff
@@ -20,24 +20,29 @@
 
 namespace souffle {
 
-/**
```

Comment:
Is the deleted comment not still accurate? It seems more descriptive than 'Literal'